README.md

```diff
@@ -1,16 +1,188 @@
-# UCAN Invocation
+# UCAN Invocation Specification v0.1.0
 
 ## Editors
 
-* _
+* [Brooklyn Zelenka](https://github.com/expede/), [Fission](https://fission.codes/)
 
 ## Authors
 
-* _
+* [Brooklyn Zelenka](https://github.com/expede/), [Fission](https://fission.codes/)
+* [Irakli Gozalishvili](https://github.com/Gozala), [DAG House](https://dag.house/)
 
-# 0. Abstract
+## Depends On
+
+* [`ucan-ipld`](https://github.com/ucan-wg/ucan-ipld/)
+
+# 0 Abstract
 
 ## Language
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
 
+# 1 Introduction
+
+## Separation
+
+Why separate the UCAN from the invocation format? Surely the UCAN itself already contains all the infomation required.
+
+I would argue that this is not the case for a few reasons.
+
+1. Authority is not intent to execute. Authority transfer is first-class in UCAN, other actions are not
+2. Mixing the two levels means that invocation info will live with the token (as a proof) on subdelegations
+3. Other detail, such as scheduling
+
+A job request may be picked up by anyone, PRIOR to UCAN delegation. A handshake or matchmaking may need to be performed.
+
+## Authority Is Not Intention
+
+Or to put it another way: "just because you can, doens't mean you should". Granting a UCAN to a peer means that they are allowed to perform some actions for a period of time. This is a feature not a bug, but also says nothing about the intention of when it should be run. I may grant a collaborative process the ability to perform actions on an ongoing basis (hmm, but vioating POLA)
```

Comment:
It is interesting point. However so far my experience has been that intention is fairly clear based on the actor you delegate capability to.

Can you think of the case where same actor may be delegated some capability just because and in other cases asked to perform an invocation ?

Reply:
Sure: Alice delegates to Bob the ability to send newsletters on a daily basis (as a service) on a regular schedule. She later wants to send an additional newsletter, not on the regular schedule.

By wrapping in an invocation, we make clear the intention of me passing this credential to you.

Using a softer analogy, if I give you a movie ticket, I'm not telling you to go to the theater immedietly; I'm letting you go "at some point".

To make clear "when" or "how" I want something run, I need to put more information on the channel. We can embed that directly in the UCAN, but it's usually orthogonal to the capability itself. Today this is often informally captured in (e.g.) an HTTP method or similar, not in the capability itself.

Reply:
> intention is fairly clear based on the actor you delegate capability to

Or maybe to rephrase: you're saying roughly "you can tell from context". Is that right?

Reply:
> Or maybe to rephrase: you're saying roughly "you can tell from context". Is that right?

Correct.

I also have since realized that we do actually signal invocation by putting invocation delegations in CAR roots. In retrospect it’s just a poor signaling so I’m convinced & rest my case